app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
+    def __init__(self, favourite_dish: str) -> None:
+        self.nickname = None
+        self._favourite_dish = favourite_dish
+
+    @abstractmethod
+    def eat_favourite_dish(self):
```

Comment:
The `eat_favourite_dish` method is marked as abstract but contains an implementation. Abstract methods should not have implementations in abstract classes. Consider removing the implementation from this method.